tests/backup/backup_upgrade_test.go

```diff
@@ -0,0 +1,29 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+)
+
+// This testcase verifies Px Backup upgrade
+var _ = Describe("{UpgradePxBackup}", func() {
+	var ()
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("UpgradePxBackup", "Upgrading backup", nil, 0)
+	})
+	It("Basic Backup Creation", func() {
+		Step("Upgrade Px Backup", func() {
+			log.InfoD("Upgrade Px Backup to version %s", latestPxBackupVersion)
+			err := UpgradePxBackup(latestPxBackupVersion)
+			log.FailOnError(err, "Failed during upgrade")
+			log.InfoD("Upgrade process complete")
```

Comment:
How about dash.VerifyFatal(err, nil, "Upgrading Px-Backup")?

Reply:
We are using `log.FailOnError` for validating an error. That is a general rule that we need to follow instead of using `dash.VerifyFatal`